packages/ui/src/proposals/modals/AddNewProposal/AddNewProposalModal.tsx

```diff
@@ -24,6 +24,7 @@ import { TextMedium, TokenValue } from '@/common/components/typography'
 import { BN_ZERO } from '@/common/constants'
 import { camelCaseToText } from '@/common/helpers'
 import { useCurrentBlockNumber } from '@/common/hooks/useCurrentBlockNumber'
+import { useKeyring } from '@/common/hooks/useKeyring'
```

Comment:
Same thing for this file since the validation doesn't happens in the `useYupValidationResolver` anymore